Doc/whatsnew/3.11.rst

```diff
@@ -2701,4 +2701,29 @@ Removed
   (Contributed by Inada Naoki in :issue:`44029`.)
 
 
+Notable changes in 3.11.4
+=========================
+
+tarfile
+-------
+
+* The extraction methods in :mod:`tarfile`, and :func:`shutil.unpack_archive`,
```

Comment:
Perhaps an item list is not needed for a single item.

Reply:
This came up in the 3.12 What's New, and we decided to keep it so the indent is consistent with the others, and it looks better as you scroll down.